### PR TITLE
[ReactNative][oss] Update PR and issue templates and guidelines

### PR DESCRIPTION
**Pull request by mkonicek**
- Simplified the PR and Issue template to make them easier to read.
- Add a very simple guide for reviewing PRs (cc @bestander, feel free to add to this)

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -2,30 +2,20 @@ Hey there and thank you for using React Native!
...
-Please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/react-native) with the tag `react-native`. StackOverflow is amazing for Q&A. Because of the reputation system it is likely the community will see and answer your question there.
-
-Make sure you **include a short code snippet to demonstrate the problem**.
-
-Have an idea for a feature?
-------------------------------------
-Post the feature request on [Product Pains](https://productpains.com/product/react-native/). It has a voting system that lets us focus on the most important issues the community is experiencing. The higher number of votes the more likely it is a feature will get implemented. You could always implement it yourself, too.
-
-None of the above, want to create a GitHub issue
+None of the above, want to create a bug report
```
Comment (bestander) [inline ISSUE_TEMPLATE.md]: that does not sound English to me

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -2,30 +2,20 @@ Hey there and thank you for using React Native!
...
 ------------------------------------------------------------------
 
 Make sure to add **all the information needed to understand the issue** so that someone can help. If the info is missing we'll add the 'Needs more information' label and close the issue until there is enough information.
 
-- If this is a bug, provide a **minimal code snippet** that reproduces it
-- Provide **screenshots** where appropriate
-- What's the **version** of React Native you're using?
-- Does this occur on iOS, Android or both?
-- Are you using Mac, Linux or Windows?
+- [ ] Provide a **minimal code snippet** that reproduces the bug
```
Comment (bestander) [inline ISSUE_TEMPLATE.md]: Should we ask them to create a project in github or better https://rnplay.org?

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
-Example: When saying "Add a function to do X", explain why it is necessary to have a way to do X.
+Example: When "Adding a function to do X", explain why it is necessary to have a way to do X.
 
 **Test plan (required)**
 
-Describe exact steps you've done to test that the code works. This means exact commands you ran, screenshots / videos if the pull request changes UI.
+Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.
 
-Make sure tests pass on Circle CI.
+Make sure tests pass on both Travis and Circle CI.
```
Comment (bestander) [inline PULL_REQUEST_TEMPLATE.md]: I am going to post about this with pictures.
Then we could link to the post

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -2,30 +2,20 @@ Hey there and thank you for using React Native!
...
+- [ ] Provide **screenshots** where appropriate
```
Comment (bestander) [inline ISSUE_TEMPLATE.md]: We should provide a sample GitHub repo for people to fork and quickly hack and example.

**Comment by bestander** [comment]
I think it is already an improvement, we should merge it.
Only one nit `None of the above, want to create a bug report`.

The other points we could do in next iterations

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (mkonicek) [inline ISSUE_TEMPLATE.md]: That's even better, will add it. Pasting code to GitHub is sometimes sufficient.

File: PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (mkonicek) [inline PULL_REQUEST_TEMPLATE.md]: Cool, has to be a public post though.

File: ISSUE_TEMPLATE.md (hunk 4 above)
Comment (mkonicek) [inline ISSUE_TEMPLATE.md]: They should add an example to the UIExplorer.

**Comment by mkonicek** [comment]
@bestander Fixed to "None of the above, create a bug report".

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**